packages/ember-routing/lib/system/router.js

```diff
@@ -585,7 +585,7 @@ const EmberRouter = EmberObject.extend(Evented, {
       seen[name] = true;
 
       if (!handler) {
-        let DefaultRoute = routeOwner._lookupFactory('route:basic');
+        let DefaultRoute = routeOwner.factoryFor('route:basic').class;
```

Comment:
similar to the above `controller:basic` example, if injections existed on `route:basic` at this point they would be lost, which I believe is a breaking change.